src/components/navigation-bar/navigation-bar.style.ts

```diff
@@ -56,6 +56,7 @@ const StyledNavigationBar = styled.nav<StyledNavigationBarProps>`
 
   ${({ navigationType, theme }) => css`
     min-height: 40px;
+    z-index: ${theme.zIndex.modal - 1};
```

Comment:
Let's create a `theme.zIndex.nav` variable for this, so we can work towards setting all the z indexes in one place